src/map.c

```diff
@@ -2023,6 +2033,33 @@ static void map_setMinimal( unsigned int wid, int value )
    window_buttonCaption( wid, "btnMinimal", (value) ? _("Normal View") : _("Minimal View") );
 }
 
+/**
+ * @brief Mark current system.
+ */
+static void map_buttonMarkSystem( unsigned int wid, const char* str )
+{
+   (void) wid;
+   (void) str;
+   StarSystem *sys;
+   if (map_selected != -1) {
+      sys=system_getIndex( map_selected );
+
+      /* Remove old note */
+      if (sys->note != NULL) {
+         free(sys->note);
+         sys->note = NULL;
+      }
+
+      /* Switch marking */
+      if (sys_isFlag(sys, SYSTEM_PMARKED))
+         sys_rmFlag(sys, SYSTEM_PMARKED);
+      else {
+         sys->note=dialogue_input(_("System note"), 0, 60, _("You can write note to this system"));
```

Comment:
How about setting text to something like `"Write a note about this system."`?

Reply:
That sounds much better